README.md

```diff
@@ -36,5 +36,7 @@ Do not set it as hidden if you want your members to see it.
 Once you have a category up and running, click on the little pencil next to Roles (Edit Categories). On the right area appear all the roles Roleypoly can manage. Drag those you want to be pickable in the category/ies then hit [Save Changes] once more.  
 And… all done!
 
+**Only people who authenticate with a Discord account that has the `Manage Roles` privilege can edit the Roleypoly page.**
```

Comment:
It's explicitly "Manage Roles" and "Administrator"

Reply:
Just fixed it (see [this commit](https://github.com/roleypoly/community-docs/pull/4/commits/2ea9b33b86fe4a92e00186cbfef03bffb0fc834c)), thank you.

Reply:
What do you mean by Discord accounts not being able to have permissions, but members having the ability? Wouldn't the opposite be true; e.g. that my privileged account has permissions, but my unprivileged account doesn't, but I am the member that has both accounts?

Reply:
A "member" is a "user account" in a "server" pretty directly.

Possibly, do we instead say "Users that have a role with Manage Roles and/or Admin permissions"?

Reply:
What about `Only users with the Administrator or Manage Roles permissions within the Discord server can edit the Discord server's Roleypoly page.`? Clearly states that it's a Discord "server" rather than any form of real server and I think communicates the message.

Reply:
Commenting again just to try to get this in. I'm not picky about the wording, I just care about it being clarified.